src/Microsoft/Silk.NET.Direct2D/Structs/D2D1BitmapRenderTargetVtblExtensions.gen.cs

```diff
@@ -257,66 +257,6 @@ public static unsafe int CreateBitmapFromWicBitmap(this ComPtr<ID2D1BitmapRender
         return ret;
     }
 
-    /// <summary>To be documented.</summary>
-    public static unsafe int CreateBitmapFromWicBitmap(this ComPtr<ID2D1BitmapRenderTarget> thisVtbl, ref IWICBitmapSource wicBitmapSource, [Flow(Silk.NET.Core.Native.FlowDirection.In)] BitmapProperties* bitmapProperties, ID2D1Bitmap** bitmap)
```

Comment:
This is fine as this is new code but worth looking at. We don't have bindings to WIC which is why this is being removed because the IWICBitmapSource is forward declared. WIC, DirectWrite, and Direct2D have a ridiculous amount of circular dependencies - should we just bundle them all into the D2D package?

Reply:
WIC will be added at a later (probably post 2.17) date